doc/dev/getting-started.rst

```diff
@@ -63,176 +63,102 @@ One-time Preparation
 
 #. Install the dependencies.
 
-   OS X users
+   i. Mac users
 
-   a.  Install Xcode from the `Mac App Store (requires root)
-       <https://developer.apple.com/xcode/>`_.
-   #.  `Register as an Apple Developer
-       <https://developer.apple.com/register>`__.
-   #.  Install the Xcode command-line tools: Xcode -> preferences ->
-       Downloads -> Command Line Tools (requires root).
+      a.  Install Xcode with::
 
-   Linux users
+             xcode-select --install
 
-   a.  Install the python development environment, virtualenv, pip, gcc, and
-       g++.
+      #.  Install `Homebrew <http://brew.sh/>`__ with::
 
-       On recent Debian and Ubuntu this can be done with::
+             /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-           sudo apt-get install python2.7-dev python-virtualenv python-pip gcc \
-           g++ git astyle gcovr cppcheck enchant
+      #.  Install the python development environment and some additional development packages::
 
-       For RHEL6::
+             brew install python astyle gcovr cppcheck enchant
+             sudo pip install --upgrade pip virtualenv
 
-           sudo yum install -y python-devel python-pip git gcc gcc-c++ make enchant
-           sudo pip install virtualenv
+   ii. Linux users
```

Comment:
In the [preview](https://github.com/dib-lab/khmer/blob/a3c24bcea16f1583913a88830c237f9a2540ad1c/doc/dev/getting-started.rst) you get a weird grey vertical bar, not sure I can spot the formatting that lead to it :-/